Makefile

```diff
@@ -140,17 +141,25 @@ ui:
 .PHONY: install
 install: check-dcos
 	$(DCOS_CMD) package install --yes marathon-lb;\
-	$(DCOS_CMD) package install --yes kubernetes --package-version=$(KUBERNETES_FRAMEWORK_VERSION) --options="$(PATH_TO_PACKAGE_OPTIONS)";\
-	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json"
+	$(DCOS_CMD) marathon app add "$(PWD)/.deploy/kubeapi-proxy.json" || true
```

Comment:
Why `|| true`? Also, I think it is preferable to do this only after the Kubernetes cluster is up.

Reply:
Yeah, I agree with @bmcstdio here. It is preferable to do this after the cluster is gone.

Reply:
Sure we can do this after, BTW the || true was put there cause of the exit status 1 it produces when it completes which kept causing my trials to fail.